janMarkusMilan/assignments/heaps/lc295/lc295.js

```diff
@@ -7,18 +7,195 @@
 
 // If the count of numbers inserted in the class is even, the median will be the average of the middle two numbers.
 
```

Comment:
Really great, thorough overview here! Would also recommend making a note about the time and space complexity of the solution as well.